x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts

```diff
@@ -204,12 +204,12 @@ export interface LogEntryDocumentFields {
 
 const convertLogDocumentToEntry = (
   sourceId: string,
-  formatMessage: (fields: LogEntryDocumentFields) => InfraLogMessageSegment[]
+  formatLogMessage: (fields: LogEntryDocumentFields) => InfraLogMessageSegment[]
```

Comment:
__note:__ makes purpose a bit clearer and allows us to not exclude this file from i18n checks